functests/vrf/vrf.go

```diff
@@ -0,0 +1,276 @@
+				} else if ipStack == ipStackIPv4 && net.ParseIP(podClinetVRFRedOverlappingIP).To4() != nil {
+					podClientVRFBlueIPAddress = "10.255.255.1"
+					podServerVRFBlueIPAddress = "10.255.255.2"
+				} else {
+					Skip("Unpsupported protocol parameter")
+				}
+				podClientIpamConfig := fmt.Sprintf(`[{"name": "%s", "mac": "%s", "ips": ["%s/24"]}, {"name": "%s", "mac": "%s", "ips": ["%s/%s"]}]`,
+					vrfBlue.Name, "20:04:0f:f1:88:01", podClientVRFBlueIPAddress, vrfRed.Name, "20:04:0f:f1:88:02", podClinetVRFRedOverlappingIP, redVRFNetworkPrefix)
+				PodClient := redefineAsPrivileged(
+					pods.DefinePodWithNetwork(pods.DefinePodOnNode(TestNamespace, podClientNodeLabel), podClientIpamConfig))
+				podServerIpamConfig := fmt.Sprintf(`[{"name": "%s", "mac": "%s", "ips": ["%s/24"]}, {"name": "%s", "mac": "%s", "ips": ["%s/%s"]}]`,
+					vrfBlue.Name, "20:04:0f:f1:88:03", podServerVRFBlueIPAddress, vrfRed.Name, "20:04:0f:f1:88:04", podServerVRFRedOverlappingIP, redVRFNetworkPrefix)
+				PodServer := redefineAsPrivileged(
+					pods.DefinePodWithNetwork(pods.DefinePodOnNode(TestNamespace, podServerNodeLabel), podServerIpamConfig))
+				waitUntilPodCreatedAndRunning(apiclient, PodClient)
+				waitUntilPodCreatedAndRunning(apiclient, PodServer)
+				podHasCorrectVrfConfig(apiclient, PodClient.Name,
+					[]map[string]string{
+						{"vrfName": VRFBlueName, "vrfClinetIP": podClientVRFBlueIPAddress, "vrfInterface": "net1"},
+						{"vrfName": VRFRedName, "vrfClinetIP": podClinetVRFRedOverlappingIP, "vrfInterface": "net2"}})
+				podHasCorrectVrfConfig(apiclient, PodServer.Name,
+					[]map[string]string{
+						{"vrfName": VRFBlueName, "vrfClinetIP": podServerVRFBlueIPAddress, "vrfInterface": "net1"},
+						{"vrfName": VRFRedName, "vrfClinetIP": podServerVRFRedOverlappingIP, "vrfInterface": "net2"}})
+				err = pingIPViaVRF(apiclient, PodClient.Name, VRFRedName, podServerVRFRedOverlappingIP)
+				Expect(err).ToNot(HaveOccurred())
+				err = pingIPViaVRF(apiclient, PodClient.Name, VRFBlueName, podServerVRFBlueIPAddress)
+				Expect(err).ToNot(HaveOccurred())
+				err = apiclient.Pods(TestNamespace).Delete(context.Background(), PodServer.Name, metav1.DeleteOptions{GracePeriodSeconds: pointer.Int64Ptr(0)})
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					_, err := apiclient.Pods(TestNamespace).Get(context.Background(), PodServer.Name, metav1.GetOptions{})
+					return err
+				}, podWaitingTime, 5*time.Second).Should(HaveOccurred())
+
+				err = pingIPViaVRF(apiclient, PodClient.Name, VRFBlueName, podServerVRFBlueIPAddress)
+				Expect(err).To(HaveOccurred())
+				err = pingIPViaVRF(apiclient, PodClient.Name, VRFRedName, podServerVRFRedOverlappingIP)
+				Expect(err).To(HaveOccurred())
+				err = pingIPViaVRF(apiclient, PodClient.Name, "eth0", podServerVRFRedOverlappingIP)
+				Expect(err).ToNot(HaveOccurred())
+			},
+			Entry(describe(""), sameNode, ipStackIPv4),
```

Comment:
Missing description?

Reply:
No need for description. I added  needed description to DescribeTable.

Reply:
Then you could simplify the describe function. Currently it's a function generator, but if you don't need the dynamic description you can have
```
describe := func(node string, ipStack string) string {

}
```
and use it without the argument. Entry accepts a function.